translate/rus.json

```diff
@@ -243,7 +243,7 @@
 "normal":"нормальный",
 "notable games":"известные игры",
 "notation":"обозначение",
-"OctoFinal":"Раунд 8",
+"OctoFinal":"Раунд 1/8",
```

Comment:
This is optional. I prefer to see OctoFinal as '1/8 of the final' and not as 'round of 8 (actually 16) engines'.